templates/components/common/quick-search.html

```diff
@@ -2,11 +2,11 @@
     <form class="form" action="{{urls.search}}">
         <fieldset class="form-fieldset">
             <div class="form-field">
-                <label class="is-srOnly" for={{name}}>{{lang "search.quick_search.input_label"}}</label>
+                <label class="is-srOnly" for="{{name}}>{{lang 'search.quick_search.input_label'}}"</label>
```

Comment:
Could we double check escaping here?

Reply:
You are right, closing quote should be after `name`